google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -1227,33 +1258,37 @@ public void testOffsetOutOfRangeError() throws IOException, InterruptedException
     }
   }
 
+  void writeSomeRows(StreamWriter sw, List<Future<AppendRowsResponse>> futureList) {
```

Comment:
nit: change this function to be :

```
void writeRows(StreamWriter sw, numRows int, List<Future<AppendRowsResponse>> futureList) {
  for (int i = 0; i < numRows; i++) {
    sw.append(CreateProtoRows(new String[] {"aaa", "bbb", "ccc"}));
  }
}
```

So it's a bit more reusable and add meaning to it instead of using the word "some". First time I read I asked myself: "How many rows this is inserting ?"

Reply:
d.